### DISCUSSION TITLE
Material You Monochrome

### DISCUSSION THREAD

**Discussion by reality-exe** [dailydotdev/daily#1353] [Feature Requests]
This is purely just a cosmetic thing however it would be nice to implement the Material You monochrome icon so it can match other apps on a users homescreen of their android phone

**Comment by idoshamun** [comment]
Hi @reality-exe,

Can you please attach a screenshot so I can better understand what you refer to?

**Reply by reality-exe** [reply]
Hallo, yea I can.
![Screenshot_20240822-112523](https://github.com/user-attachments/assets/11d86965-496e-4c11-8e75-cd7495e3d6fc)
The monochrome I'm talking about would be the app's icon matching the Material You theme that google provides.

**Reply by idoshamun** [reply]
I sent another version to review. It should be fixed soon. Feel free to reopen in case it doesn't
